client/my-sites/email/inbox-management/mailbox-selection-list/style.scss

```diff
@@ -0,0 +1,126 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: #fff;
+}
```

Comment:
There are two issues here:
1. This use of `body` without any other selector means we could result in _all_ pages using a white background. We need something to limit this to this page. (If we move to a new section, we may be able to use `.is-section-inbox` for that.
2. We should use `var( --studio-white );` for the colour instead of `#fff`.